<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerClass</class>
 <widget class="QWidget" name="ColorPickerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorPicker</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QColorLabel1D" name="som1DLabel">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>10</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QColorLabel2D" name="som2DLabel">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>49</y>
     <width>431</width>
     <height>431</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="computeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="ColorSpaceView" name="glWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pasteButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorSpaceView</class>
   <extends>QWidget</extends>
   <header location="global">colorspaceview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QColorLabel1D</class>
   <extends>QLabel</extends>
   <header>qcolorlabel.h</header>
  </customwidget>
  <customwidget>
   <class>QColorLabel2D</class>
   <extends>QLabel</extends>
   <header>qcolorlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="colorpicker.qrc"/>
 </resources>
 <connections/>
</ui>
